protos/v1alpha1/team_link.proto

```diff
@@ -52,3 +52,32 @@ message GitHubUser {
   // Optional. Any additional information such as type of the user (employee, vendor, etc.).
   map<string, string> labels = 15;
 }
+
+// TeamMappings represents a list of destination(GitHub) to source team mappings.
+message TeamMappings {
+  // Required. GitHub to source team mappings.
+  repeated TeamMapping team_mappings = 1;
+}
+
+// TeamMapping represents a mapping from a destination(GitHub) team to a list of
+// source teams. It also contains other required information such as Github
+// organization.
+message TeamMapping {
+  // Required. The ID of the GitHub team.
+  int64 github_team_id = 1;
+
+  // Required. The ID of the GitHub organization.
+  int64 github_org_id = 2;
+
+  // Required. Source team IDs. The actual value depends on the source team
+  // service API, what value it uses to retrieve the source team.
+  repeated string source_team_ids = 3;
+
+  // Optional. The name of the GitHub team, note it is changeable and is
```

Comment:
This is interesting. I wonder what's the expected behavior if this is updated. Is the old GH team going to be removed? Not blocking

Reply:
Do you mean in GitHub, just verified, everything (ID etc) stays the same when the team name is changed.